common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -125,6 +136,8 @@ def student_view(self, context):
                 'progress_detail': Progress.to_js_detail_str(progress),
                 'type': child.get_icon_class(),
                 'id': child.scope_ids.usage_id.to_deprecated_string(),
+                'bookmarked': is_bookmarked,
+                'path': '{0} > {1}'.format(sequence_path, child.display_name)
```

Comment:
If instead of constructing the string like this, we collected the display_names in a list and used `" > ".join(display_names), it would take care of empty values, right?